### ISSUE TITLE
ansible-playbook generate-templates.yaml failed : has no attribute 'image_list'

### ISSUE THREAD

**Issue by Shea-D** [kubevirt/common-templates#443]
**Is this a BUG REPORT or FEATURE REQUEST?**: 
BUG.

**What happened**:
When I tried to generate templates, before centos stream 8, it was successful.

```
......
TASK [Load RHEL 6 versions] *************************************************************************************************************************************************************************
ok: [127.0.0.1]

TASK [Generate RHEL 6 templates] ********************************************************************************************************************************************************************
changed: [127.0.0.1] => (item={u'workload': u'server', u'tablet': False, u'default': False, u'cpus': 1, u'iothreads': False, u'emulatorthread': False, u'memsize': u'1Gi', u'flavor': u'tiny'})
changed: [127.0.0.1] => (item={u'workload': u'desktop', u'tablet': True, u'default': False, u'cpus': 1, u'iothreads': False, u'emulatorthread': False, u'memsize': u'1Gi', u'flavor': u'tiny'})
changed: [127.0.0.1] => (item={u'workload': u'server', u'tablet': False, u'default': True, u'cpus': 1, u'iothreads': False, u'emulatorthread': False, u'memsize': u'2Gi', u'flavor': u'small'})
changed: [127.0.0.1] => (item={u'workload': u'desktop', u'tablet': True, u'default': False, u'cpus': 1, u'iothreads': False, u'emulatorthread': False, u'memsize': u'2Gi', u'flavor': u'small'})
changed: [127.0.0.1] => (item={u'workload': u'server', u'tablet': False, u'default': False, u'cpus': 1, u'iothreads': False, u'emulatorthread': False, u'memsize': u'4Gi', u'flavor': u'medium'})
changed: [127.0.0.1] => (item={u'workload': u'desktop', u'tablet': True, u'default': False, u'cpus': 1, u'iothreads': False, u'emulatorthread': False, u'memsize': u'4Gi', u'flavor': u'medium'})
changed: [127.0.0.1] => (item={u'workload': u'server', u'tablet': False, u'default': False, u'cpus': 2, u'iothreads': False, u'emulatorthread': False, u'memsize': u'8Gi', u'flavor': u'large'})
changed: [127.0.0.1] => (item={u'workload': u'desktop', u'tablet': True, u'default': False, u'cpus': 2, u'iothreads': False, u'emulatorthread': False, u'memsize': u'8Gi', u'flavor': u'large'})

TASK [Load CentOS Stream 8 image urls] **************************************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'ansible.plugins.lookup.osinfo.OsInfoGObjectProxy object' has no attribute 'image_list'\n\nThe error appears to be in '/home/d/common-templates/generate-templates.yaml': line 123, column 5, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n  - name: Load CentOS Stream 8 image urls\n    ^ here\n"}

PLAY RECAP ******************************************************************************************************************************************************************************************
127.0.0.1                  : ok=9    changed=5    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

it is said that there is no attribute ‘image_list’, I have followed the steps in readme.md before I ran this command 'ansible-playbook generate-templates.yaml', and I don't know why it reported this error.

**What you expected to happen**:
generate all templates.

**How to reproduce it (as minimally and precisely as possible)**:
git clone https://github.com/kubevirt/common-templates
cd common-templates/
git submodule init
git submodule update
make -C osinfo-db
yum install jq
yum install ansible
yum install libosinfo
yum install python-gobject
yum install osinfo-db-tools
yum install intltool
ansible-playbook generate-templates.yaml

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
v0.51.0
- Kubernetes version (use `kubectl version`):
v0.21.0
- common-templates version: 
latest
- OS (e.g. from /etc/os-release):
centos7
- Kernel (e.g. `uname -a`):
Linux 3.10.0-957.el7.x86_64
- Others:
It is an okd cluster.

**Comment by ksimon1** [comment]
Hi, can you please run:
```
git pull upstream master
git submodule update -f
make generate
```
and if the same error occured again, can you please put here the whole log?

**Comment by 0xFelix** [comment]
Your libosinfo version is too old. CentOS 7 comes with libosinfo version 1.1.0, but the image_list attribute was first included in version 1.3.0. Can you generate the templates on another host?

**Comment by Shea-D** [comment]
> Your libosinfo version is too old. CentOS 7 comes with libosinfo version 1.1.0, but the image_list attribute was first included in version 1.3.0. Can you generate the templates on another host?

I'll try it! it seems the solution to this problem! thanks for reply!
